demos/profiling/src/utils.py

```diff
@@ -88,6 +95,18 @@ def postprocess(cmap, paf):
 
     return postprocess
 
+def get_distance_function(distance_function, video_width=None, video_height=None):
+    if distance_function == "keypoints_vote":
+        return create_keypoints_voting_distance(
+            keypoint_distance_threshold=DISTANCE_THRESHOLD,
+            detection_threshold=DETECTION_THRESHOLD
+        )
```

Comment:
I want to remark that the `keypoints_vote` distance function that you defined is not exactly the same as the `keypoints_distance` used in this demo, since the one used in this demo rescales the distances between the detection and the tracked_object keypoints taking into account the size of the object (so that for smaller objects, the corresponding keypoints need to be closer to each other in order to make a positive vote).